crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1059,6 +1199,36 @@ impl Command for LogComponents {
     }
 }
 
+/// [`Command`] to query an [`Entity`].
+pub struct WithQuery<Q, F = ()>
```

Comment:
This needs a public constructor, otherwise it's useless to publicly expose.

Reply:
Agreed, that's an oversight on my part! Resolved in updated commit.